### PR TITLE
[1LP][RFR] Testing multiple automation requests

### PR DESCRIPTION
**Pull request by ganeshhubale**
- Testing multiple automation requests and denying

{{ pytest: cfme/tests/infrastructure/test_rest_automation_request.py -k 'test_multiple_automation_requests' -vv --long-running }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/infrastructure/test_rest_automation_request.py [hunk 1]
```diff
@@ -257,6 +258,23 @@ def test_edit_requests(self, collection, appliance, requests_pending, from_detai
...
+        """
+        Polarion:
+            assignee: ghubale
+            initialEstimate: 1/30h
+            casecomponent: Automate
+
+        Bugzilla:
+            1723830
+        """
+        requests_data = _automation_requests_data(vm, approve=False, num=2000)
```
Comment (john-dupuy) [inline cfme/tests/infrastructure/test_rest_automation_request.py]: @ganeshhubale I see that the BZ was written when a customer submitted 60000 requests, is 2000 enough? Thoughts on parametrizing to more? e.g. `[2000, 10,000, 30,000, 60,000]`

Comment (ganeshhubale) [reply cfme/tests/infrastructure/test_rest_automation_request.py]: Thanks @john-dupuy for going through BZ.
- According to me, parametrizing this test case with different number of requests is not actually a use case. Here we want to check if huge number of automation requests provisioned are getting denied or not.
- Yes, we can increase the number of requests up to the number we want. So I updated it to 60,000 and making this PR long_running(Although this is rest test case)

Comment (mshriver) [reply cfme/tests/infrastructure/test_rest_automation_request.py]: Restarting PRT with `--long-running` included in the options, as this change caused the test to not be collected in run 33445

**Pull request merged**